### PR TITLE
Moves behaviors in microwave attackby into procs

### PR DESCRIPTION
**Pull request by FalloutFalcon**
## About The Pull Request

Sorts welder, wire cutter, and washing tools into there respective procs

## Why It's Good For The Game

We should uh use the atom/procs where they should be used i think.
Also lets you clean microwaves with things other then soap and spray bottles (rags was my goal but should apply to some other odd methods)

## Changelog

:cl:
tweak: you can clean microwaves with a rag
refactor: microwave attackby behaviors moved into smarter procs
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/kitchen_machinery/microwave.dm [hunk 1]
```diff
@@ -195,6 +163,35 @@
...
+/obj/machinery/microwave/welder_act(mob/living/user, obj/item/I)
+	. = ..()
+	if(broken == 1)
+		user.visible_message("<span class='notice'>[user] starts to fix part of \the [src].</span>", "<span class='notice'>You start to fix part of \the [src]...</span>")
+		if(I.use_tool(src, user, 20))
+			user.visible_message("<span class='notice'>[user] fixes \the [src].</span>", "<span class='notice'>You fix \the [src].</span>")
+			broken = 0
+			update_appearance()
+			return TRUE
+	return
```
Comment (west3436) [inline code/modules/food_and_drinks/kitchen_machinery/microwave.dm]: Unnecessary trailing return

File: code/modules/food_and_drinks/kitchen_machinery/microwave.dm [hunk 2]
```diff
@@ -195,6 +163,35 @@
...
+/obj/machinery/microwave/wirecutter_act(mob/living/user, obj/item/I)
+	. = ..()
+	if(broken == 2)
+		user.visible_message("<span class='notice'>[user] starts to fix part of \the [src].</span>", "<span class='notice'>You start to fix part of \the [src]...</span>")
+		if(I.use_tool(src, user, 20))
+			user.visible_message("<span class='notice'>[user] fixes part of \the [src].</span>", "<span class='notice'>You fix part of \the [src].</span>")
+			broken = 1
+			update_appearance()
+			return TRUE
+	return
```
Comment (west3436) [inline code/modules/food_and_drinks/kitchen_machinery/microwave.dm]: Trailing return

File: code/modules/food_and_drinks/kitchen_machinery/microwave.dm [hunk 3]
```diff
@@ -195,6 +163,35 @@
...
+
+/obj/machinery/microwave/wash(clean_types)
+	. = ..()
+	if(dirty != 0)
```
Comment (west3436) [inline code/modules/food_and_drinks/kitchen_machinery/microwave.dm]: if(dirty)

**Review by MarkSuckerberg — APPROVED** [review]
nice :+1:

**Pull request merged**